scr/main.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import StartScreen from './startScreen/screen.tsx';
+import './index.less';
+
+ReactDOM.createRoot(document.getElementById('root')!).render(
+  <React.StrictMode>
```

Comment:
You will need to use react router to switch between pages. Install it if not included in basic react.
https://reactrouter.com/en/main/components/routes